### PR TITLE
Add r-glmmtmb 0.2.0

### PR DESCRIPTION
**Pull request by ccwang002**
Add R package glmmTMB v0.2.0.

### CODE REVIEW

**Comment by isuruf** [comment]
Please merge your branch with master

**Comment by ccwang002** [comment]
Appveyor's test has passed as well. See https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.22953

File: recipes/r-glmmtmb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - r-nlme
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('glmmTMB')"  # [not win]
+    - "\"%R%\" -e \"library('glmmTMB')\""  # [win]
+
+about:
+  home: https
```
Comment (cbrueffer) [inline recipes/r-glmmtmb/meta.yaml]: Could you fix the homepage? Looks good otherwise.

Comment (ccwang002) [reply recipes/r-glmmtmb/meta.yaml]: Done. Thanks!

**Comment by ccwang002** [comment]
Is there any issue blocking this PR? I'd like to add some R packages depending on it.

**Comment by cbrueffer** [comment]
@ccwang002 not too many people can actually merge recipes; usually @bgruening merges mine. Maybe we can get his attention :-)

**Comment by bgruening** [comment]
I hate it if I block someones work. Sorry.

**Comment by cbrueffer** [comment]
Thanks Björn! If you have a minute, there are a few more PRs ready at https://github.com/conda-forge/staged-recipes/pulls/cbrueffer , https://github.com/conda-forge/staged-recipes/pulls/ccwang002 and https://github.com/conda-forge/staged-recipes/pulls/brisk022

**Pull request merged**